### PR TITLE
OpenSearch & Pipeline Preparation

### PR DESCRIPTION
**Pull request by KasparMinn**
This pull request is to include to separate a newly created OpenSearch initialization file  from the old one as well as fix some possible errors with the Pipeline's reading of YML files. 

The new OpenSearch init file, "init-live.sh" cleans all current OpenSearch data if it has any, and creates indices and directories that are empty, but necessary for the Training module to function. The "init-mock.sh" remains to test OpenSearch with mocks if necessary. 

- Created a new OpenSearch initialization that will work with full back-end data.
- Added a clause into Pipeline for possible extra nesting in YML files.
- Added pipe symbol replacement as a separate clause in DataMapper's string replace. 
- Added Liquibase's master changelog file.
- Fixed one last_modified YML.

### CODE REVIEW

**Review by ExiRain — APPROVED** [review]

File: DSL/DataMapper/js/util/stringReplace.js
Comment (MarkoKull) [inline DSL/DataMapper/js/util/stringReplace.js:1]: It creates json in 'else' block, but not when replacing that pipe sign. Is that ok?

And just asking -  does it always related to 'example' block only?

File: DSL/OpenSearch/init-live.sh
Comment (MarkoKull) [inline DSL/OpenSearch/init-live.sh:1]: Would be nice to define the host and port in the beginning of the script. Good script otherwise.

**Review by MarkoKull — COMMENTED** [review]
Added a question about stringReplace.js.

File: DSL/DataMapper/js/util/stringReplace.js
Comment (KasparMinn) [reply DSL/DataMapper/js/util/stringReplace.js:1]: Fixed the json data creation, second should always be related to the block if it's pipe specifically that needs to be replaced. There's no other place in the application that it's requested.

**Review by MarkoKull — APPROVED** [review]

**Pull request merged**